libjpeg_addons/src/libjpg_addons.c

```diff
@@ -0,0 +1,294 @@
+/*
+# PS2 JPEG Library Port
+# Written by linuzappz.
+# Modified by ntba2 for myPS2.
+# Added jpgOpenFILE by Polo for ULE
+*/
```

Comment:
The interface is compatible with the old library, but the implementation is all yours. This comment needs to be updated or removed.

Reply:
Not really, is a mixed solution.
I have used some logic from previous implementation and some created by me now.
My original idea was to create exactly the same implementation and headers than before, but the way it was used before ....it didn't have any sense at all.
So in order to don't duplicate code in OPL and wLE, I decided to create something quite compatible with legacy "contract" but easier to use.